int32 vehicle_id

float64 x
float64 y

string message